src/librustc/middle/infer/combine.rs

```diff
@@ -354,50 +356,55 @@ pub trait Combineable<'tcx> : Repr<'tcx> + TypeFoldable<'tcx> {
 impl<'tcx,T> Combineable<'tcx> for Rc<T>
     where T : Combineable<'tcx>
 {
-    fn combine<C:Combine<'tcx>>(combiner: &C,
-                                a: &Rc<T>,
-                                b: &Rc<T>)
-                                -> cres<'tcx, Rc<T>>
+    fn combine<C:Combine<'tcx>>(
+               combiner: &C,
+               a: &Rc<T>,
+               b: &Rc<T>)
+               -> cres<'tcx, Rc<T>>
```

Comment:
the style guide says to do this http://aturon.github.io/style/whitespace.html#line-wrapping. And I've seen some cases in librustc.